public/content/documentation/native/controls/link.md

```diff
@@ -0,0 +1,325 @@
+      - `FocusRequester.createRefs()` helps to request focus to inner elements with in the group
+
+### State
+
+   - **Android Views**
+      - Active: `android:enabled=true`
+      - Disabled: `android:enabled=false`
+      - Announcement: disabled
+
+   - **Android Compose**
+      - Active: default state is active and enabled. Use `RadioButton(enabled = true)` to specify explicitly
+      - Disabled:  `RadioButton(enabled = false)` announces as disabled
+      - Alternatively can use `modifier = Modifier.semantics { disabled() }` to announce as disabled
+      - Use `modifier = Modifier.semantics { stateDescription = "" }` to have a customized state announcement
+
+### Focus
+
+   - Only manage focus when needed. Primarily, let the device manage default focus order
+   - Consider how focus should be managed between child elements and their parent views or containers
+   - External keyboard tab order often follows the screen reader focus, but sometimes needs focus management
+   - Initial focus on a screen should land in a logical place, such as back button, screen title, first text field, or first heading
+   - When a link is closed, the focus should return to the triggering element
+
+   - **Android Views**
+      - `importantForAccessibility` makes the element visible to the Accessibility API
+      - `android:focusable`
+      - `android=clickable`
+      - Implement an `onClick( )` event handler for keyboard, as well as `onTouch( )`
+      - `nextFocusDown`
+      - `nextFocusUp`
+      - `nextFocusRight`
+      - `nextFocusLeft`
+      - `accessibilityTraversalBefore` (or after)
+      - To move screen reader focus to newly revealed content: `Type_View_Focused`
+      - To NOT move focus, but dynamically announce new content: `accessibilityLiveRegion`(set to polite or assertive)
+      - To hide controls: `Important_For_Accessibility_false`
+      - For a `ViewGroup`, set `screenReaderFocusable=true` and each inner object’s attribute to keyboard focus (`focusable=false`)
+   
+   - **Android Compose**
+      - `Modifier.focusTarget()` makes the component focusable
+      - `Modifier.focusOrder()` needs to be used in combination with FocusRequesters to define focus order
+      - `Modifier.onFocusEvent()`, `Modifier.onFocusChanged()` can be used to observe the changes to focus state
+      - `FocusRequester` allows to request focus to individual elements with in a group of merged descendant views
+      - *Example:* To customize the focus events behaviour or the sequence of focus,
+         - step 1: define the focus requester prior. `val (first, second) = FocusRequester.createRefs()`
+         - step 2: update the modifier to set the order. `modifier = Modifier.focusOrder(first) { this.down = second }`
+         - focus order accepts following values: up, down, left, right, previous, next, start, end
+         - step 3: use `second.requestFocus()` to gain focus
+
+### Code Example
+
+   - **Jetpack Compose**
+```html
```

Comment:
we should specify this as java or kotlin and then we need to add a task to add some more highlighting for this in the styles

Reply:
When I tried to edit this, it wouldn't build correctly, so I left out this edit from my updated PR. Can we pair Monday on this to discuss?